client/signup/steps/import-from/components/importer-drag/index.tsx

```diff
@@ -41,10 +42,11 @@ interface Props {
 	importerStatus: ImportJob;
 	importerData: ImporterConfig;
 	site: Site;
+	urlData: UrlData;
 	startImport: ( siteId: number, type: string ) => void;
 }
 const ImporterDrag: React.FunctionComponent< Props > = ( props ) => {
-	const { importerStatus, importerData, site /*, startImport*/ } = props;
+	const { importerStatus, importerData, site, urlData /*, startImport*/ } = props;
```

Comment:
Why is `startImport` commented out?

Reply:
It is not needed now. Maybe shortly soon when we make a final optimization.